src/backend_manager.h

```diff
@@ -168,7 +173,12 @@ class TritonBackendManager {
  private:
   DISALLOW_COPY_AND_ASSIGN(TritonBackendManager);
   TritonBackendManager() = default;
+
+  Status PreloadBackend(
+      const std::string& backend_name,
+      const triton::common::BackendCmdlineConfigMap& config_map);
   std::unordered_map<std::string, std::shared_ptr<TritonBackend>> backend_map_;
+  std::list<std::shared_ptr<TritonBackend>> persist_backends_;
```

Comment:
After another look at the BackendManager, actually it doesn't even unload the backend once loaded, probably to avoid re-loading the framework libraries which may cause issue with static initialization. So may not even need this: `CreateBackend()` will put the loaded backend into `backend_map_`.

Reply:
Good point. Removed.